benchexec/tools/muval.py

```diff
@@ -0,0 +1,89 @@
+"""
+BenchExec tool-info module for MuVal.
+
+MuVal is a verification tool based on modal μ-calculus for C programs.
+"""
+
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
```

Comment:
Please add a function called `project_url()` that returns the URL of your project.

Reply:
The BitBucket URL that is now linked is broken (error 404).